Client/Frontend/Browser/MailProviders.swift

```diff
@@ -85,19 +125,21 @@ class MyMailIntegration: MailProvider {
     ]
 
     func newEmailURLFromMetadata(_ metadata: MailToMetadata) -> URL? {
-        return constructEmailURLString(beginningScheme, metadata: metadata, supportedHeaders: supportedHeaders).asURL
+        return constructEmailURLString(urlFormat,
+                                       metadata: metadata,
+                                       supportedHeaders: supportedHeaders).asURL
     }
 }
 
 class MailRuIntegration: MyMailIntegration {
     override init() {
         super.init()
-        self.beginningScheme = "mailru-mailto://?"
+        self.urlFormat = "mailru-mailto://?%@&%@"
```

Comment:
Does this expect any string? I am wondering if this can be exploited in any way in terms of security with malicious input.

Reply:
Yes, this is expecting a string. I don't think it could be exploited on our end but I am not 100% sure either.